mod/admin.php

```diff
@@ -1351,8 +1355,6 @@ function admin_page_site(App $a)
 	if ($optimize_max_tablesize <= 0) {
 		$optimize_max_tablesize = -1;
 	}
-	// Default list of forbidden names, classic role names from RFC 2142
```

Comment:
How do we ensure now to not use these names?

Reply:
They are in the default value for the config key.